app/policies/application_policy.rb

```diff
@@ -36,6 +36,20 @@ def destroy?
     false
   end
 
+  def query?
```

Comment:
@hsong-rh Can't this be show?

Reply:
This method is for `graphql` only. It will not display in `metadata`

Reply:
Removed